docs/release-notes/3.0.0_Alpha.md

```diff
@@ -0,0 +1,91 @@
+
+For detailed instructions on deploying rootless PMM, check the [Setting up PMM Server](https://docs.percona.com/percona-monitoring-and-management/setting-up/index.html#set-up-pmm-server) topic.
+
+### Transition to Oracle Linux 9 completed
+
+With the release of PMM 3, we are completing the transition away from CentOS/RHEL 7-based builds and fully embracing Oracle Linux 9 as the default operating system for PMM-base images. This transition began with PMM 2.38 and is now being finalized for all users, including those who had upgraded from 2.37 and earlier versions using the **PMM Upgrade** option on the Home Dashboard.
+
+By leveraging Oracle Linux 9 as the default operating system, we are ensuring a more stable, updatable and well-maintained operating system for PMM's base images.
+
+## Improved UX
+
+### Grafana upgraded to latest version
+
+PMM now integrates Grafana 10.4, which delivers the following important enhancements alongside all the advancements introduced since the previous Grafana 9.2.20 integration. For the full list of changes included in this update, see [Grafana's 10.4 release blog](https://grafana.com/blog/2024/03/06/grafana-10.4-release-all-the-latest-features/).
+
+**Improved navigation**
+With the 10.4 update, PMM now includes a revamped header with search, breadcrumbs, and a reorganized menu that groups related tools together, making it easier to navigate PMM's features and find what you need:
+
+![New Dashboard page](../_images/Navigation_Dashboard.png)
+
+**Improved Alerting workflow**
+Leveraging the new Grafana user interface updates, we've taken the opportunity to refine the workflow for creating alerts from the **Alert Rules** and **Alert Rule Templates** pages.
+
+You'll notice separate, more visible options for creating different types of alert rules, cutting down on unnecessary steps and making it easier to manage various alert rules, templates, and configurations:
+
+![New Alerts rule page](../_images/Grafana_Alerting.png)
+
+***One central location for all Administration settings**
+
+All PMM-related configuration and inventory options have been relocated from the **Configuration** section to a unified **Administration** menu.
+Grouping all administration-related tools means that you get easier, centralized access to all PMM configuration settings, making it easier to find what you need.
+
+![Administration menu](../_images/Admin_menu.png)
+
+For more details about all the changes coming Grafana v10.4, see [Grafana's release blog](https://grafana.com/blog/2024/03/06/grafana-10.4-release-all-the-latest-features/).
+
+## Enhanced PMM deployment on AMI and OVF environments
+
+PMM is now deployed inside containers within AMI and OVF instances. These containers operate in a rootless mode and are managed via Podman. This approach significantly increases security for a more robust and secure monitoring of your environments.
+
+## Finalized DBaaS migration to Percona Everest
```

Comment:
let's add here about seamless integration from everest to PMM when you use pmm for monitoring (this  should be already in latest version of Everest )